coredis/commands/core.py

```diff
@@ -2050,7 +2050,9 @@ async def hmset(self, key: KeyT, field_values: Dict[StringT, ValueT]) -> bool:
         readonly=True,
         group=CommandGroup.HASH,
     )
-    async def hmget(self, key: KeyT, fields: Parameters[StringT]) -> Tuple[AnyStr, ...]:
+    async def hmget(
+        self, key: KeyT, fields: Parameters[StringT]
+    ) -> Tuple[Optional[AnyStr], ...]:
```

Comment:
Could you add a test for this - since the tests are run with type checking we should get extra validation.

Reply:
done

Reply:
Just for the record, I lied. Since the response type is `Tuple[AnyStr, ...]` and the fixtures in the tests aren't properly typed to `bytes` or `str`.. `AnyStr` ~= `Any`. I'll follow this up soon.

Reply:
😨